### PR TITLE
5630 add documentation for recovery device feature

### PR DESCRIPTION
**Pull request by FirelightFlagboy**
You can take a look at the [Recovery device](https://parsec-cloud--5655.org.readthedocs.build/en/5655/userguide/recovery_device.html) page

### CODE REVIEW

File: docs/userguide/recovery_device.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+   .. image:: screens/recovery_device_create_device_confirmation.png
+      :alt: Create a recovery device confirmation modal
+
+   .. note::
+
+      The passphrase should be handled like any password.
+
+      You could print out the passphrase and put-it in a safe to secure it.
+      Nevertheless, you should not save it alongside the recovery device.
```
Comment (mmmarcos) [inline docs/userguide/recovery_device.rst:53]:
```suggestion
      The passphrase should be handled carefully like any other password.

      You can print out the passphrase and put-it in a safe to secure it.
      Nevertheless, you should not save it alongside the recovery file.
```

File: docs/userguide/recovery_device.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+
+Use a recovery device
+#####################
+
+1. At that point, you select ``Recover a Device``, that will open a new modal:
```
Comment (mmmarcos) [inline docs/userguide/recovery_device.rst:59]:
```suggestion
1. In the Device recovery dialog, select ``Recover a Device``
```

File: docs/userguide/recovery_device.rst [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+   .. image:: screens/recovery_device_recover.png
+      :alt: Recover a device modal
+
+2. From here:
+
+   a. You select the recovery device's file that correspond to the device you want to recover.
+   b. Enter the passphrase for the recovery device.
+   c. Enter the desired device name.
```
Comment (mmmarcos) [inline docs/userguide/recovery_device.rst:68]:
```suggestion
2. From here:

   a. Select the recovery file of the device you want to recover.
   b. Enter the passphrase for the recovery device.
   c. Enter a new name for the device.

   .. image:: screens/recovery_device_recover.png
      :alt: Recover a device modal
```

File: docs/userguide/recovery_device.rst [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+3. Click next, you will be prompted to enter a new password and confirm it.
+
+   .. image:: screens/recovery_device_recover_new_password.png
+      :alt: Enter the new password for the recovered device modal
+
+4. The last modal is a confirmation prompt indicating everything went fine, you can click on continue.
```
Comment (mmmarcos) [inline docs/userguide/recovery_device.rst:75]:
```suggestion
4. A message is displayed to confirm the device has been recovered, you can click `Continue`.
```

File: docs/userguide/recovery_device.rst [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+   .. image:: screens/recovery_device_recover_confirmation.png
+      :alt: Device recovered confirmation modal
+
+5. You will find the recovered device in the devices' list
```
Comment (mmmarcos) [inline docs/userguide/recovery_device.rst:80]:
```suggestion
5. You can now access Parsec as usual using the new recovered device from the devices' list
```

**Review by mmmarcos — CHANGES_REQUESTED** [review]
Thanks for this docs! Some suggestions below.

Also, can you add a `newsfragment` for this so it will appear in the changelog?

If possible, it will be nice to also remove blur from images. It is more user friendly and consistent with other sections of the docs, like [Create new devices](https://docs.parsec.cloud/en/latest/userguide/new_device.html), using test data instead of blurring images.

File: docs/userguide/index.rst [hunk 6]
```diff
@@ -16,6 +16,7 @@ User Guide
     new_device
     share_data
     revoke_user
+    recovery_device
```
[the remark this answers is not in the document]
Comment (FirelightFlagboy) [reply docs/userguide/index.rst]: done

File: newsfragments/5630.doc.rst [hunk 7]
```diff
@@ -0,0 +1,2 @@
+Add documentation for the ``recovery device`` feature.
+That is an old feature that wasn't documented until now.
```
Comment (mmmarcos) [inline newsfragments/5630.doc.rst:2]: What about:
```suggestion
This feature was introduced in v2.6.0 and it was not documented until now.
```

**Review by mmmarcos — APPROVED** [review]
:+1:

File: docs/locale/fr/LC_MESSAGES/userguide/recovery_device.po [hunk 8]
```diff
@@ -21,145 +21,179 @@ msgstr ""
...
 #: ../../userguide/recovery_device.rst:9
 msgid ""
 "A *recovery device* is a backup of an existing device. If you lose access to "
 "Parsec (you forget the password), the recovery device will allow you to "
 "recreate a new device and set a new password."
 msgstr ""
+"Un *appareil de récupération* est une sauvegarde d'un appareil existant. Si "
+"vous perdez accès à Parsec (vous avez oublié votre mots de passe), "
+"L'appareil de récupération vous permet de recréer un nouvel appareil avec un "
+"nouveau mots de passe."
```
Comment (mmmarcos) [inline docs/locale/fr/LC_MESSAGES/userguide/recovery_device.po:35]:
```suggestion
"nouveau mot de passe."
```

File: docs/locale/fr/LC_MESSAGES/userguide/recovery_device.po [hunk 9]
```diff
@@ -21,145 +21,179 @@ msgstr ""
...
+msgstr "Dialogue de création d'un appareil de récupération"
 
 #: ../../userguide/recovery_device.rst:38
 msgid ""
-"Click next, you will be prompted to enter the password for the device you "
-"have selected"
+"Click ``Next``, you will be prompted to enter the password for the device "
+"you have selected"
 msgstr ""
+"Cliquer sur ``Suivant``, vous serez demandé d'entrer le mots de passe pour "
```
Comment (mmmarcos) [inline docs/locale/fr/LC_MESSAGES/userguide/recovery_device.po:107]:
```suggestion
"Cliquer sur ``Suivant``, vous serez demandé d'entrer le mot de passe pour "
```

File: docs/locale/fr/LC_MESSAGES/userguide/recovery_device.po [hunk 10]
```diff
@@ -21,145 +21,179 @@ msgstr ""
...
 
 #: ../../userguide/recovery_device.rst
 msgid "Create a recovery device confirmation modal"
-msgstr ""
+msgstr "Dialogue de confirmation de création d'appareil de récupération"
 
 #: ../../userguide/recovery_device.rst:50
 msgid "The passphrase should be handled carefully like any other password."
 msgstr ""
+"La phrase de passe doit être manipulée comme n'importe quels mots de passe."
```
Comment (mmmarcos) [inline docs/locale/fr/LC_MESSAGES/userguide/recovery_device.po:132]:
```suggestion
"La phrase de passe doit être manipulée comme n'importe quels mot de passe."
```

File: docs/locale/fr/LC_MESSAGES/userguide/recovery_device.po [hunk 11]
```diff
@@ -21,145 +21,179 @@ msgstr ""
...
 
 #: ../../userguide/recovery_device.rst:52
 msgid ""
 "You can print out the passphrase and put-it in a safe to secure it. "
 "Nevertheless, you should not save it alongside the recovery file."
 msgstr ""
+"Vous pouvez imprimer la phrase de passe et la mettre dans un coffre-fort "
+"pour la sécuriser. Néanmoins, vous ne devez pas la sauvegarder à coter du "
```
Comment (mmmarcos) [inline docs/locale/fr/LC_MESSAGES/userguide/recovery_device.po:140]:
```suggestion
"pour la sécuriser. Néanmoins, vous ne devez pas la sauvegarder à coté du "
```

File: docs/locale/fr/LC_MESSAGES/userguide/recovery_device.po [hunk 12]
```diff
@@ -21,145 +21,179 @@ msgstr ""
...
 msgid "Recover a device modal"
-msgstr ""
+msgstr "Dialogue de récupération d'appareil"
 
 #: ../../userguide/recovery_device.rst:70
 msgid ""
-"Click next, you will be prompted to enter a new password and confirm it."
+"Click ``Next``, you will be prompted to enter a new password and confirm it."
 msgstr ""
+"Cliquer sur ``Suivant``, vous serez demandé d'entrer un nouveau mots de "
```
Comment (mmmarcos) [inline docs/locale/fr/LC_MESSAGES/userguide/recovery_device.po:175]:
```suggestion
"Cliquer sur ``Suivant``, vous serez demandé d'entrer un nouveau mot de "
```

File: docs/locale/fr/LC_MESSAGES/userguide/recovery_device.po [hunk 13]
```diff
@@ -21,145 +21,179 @@ msgstr ""
...
+"passe et de le confirmer."
 
 #: ../../userguide/recovery_device.rst
 msgid "Enter the new password for the recovered device modal"
 msgstr ""
+"Dialogue de configuration du nouveau mots de passe pour l'appareil récupéré"
```
Comment (mmmarcos) [inline docs/locale/fr/LC_MESSAGES/userguide/recovery_device.po:181]:
```suggestion
"Dialogue de configuration du nouveau mot de passe pour l'appareil récupéré"
```

**Review by mmmarcos — CHANGES_REQUESTED** [review]

**Comment by mmmarcos** [comment]
There is some legacy entries that should be removed from the `.po` file:
* `#: ../../userguide/recovery_device.rst`: these entries do not include the line number and do not appear in the rst file
* `#: ../../userguide/recovery_device.rst:20`: these entries should be kept

**Comment by FirelightFlagboy** [comment]
> `#: ../../userguide/recovery_device.rst`: these entries do not include the line number and do not appear in the rst file

Those texts comes from the `alt` of images. they would come back when you run `make intl-fr`

> `#: ../../userguide/recovery_device.rst:20`: these entries should be kept

I don't understand, we only have 1 entry for `../../userguide/recovery_device.rst:20`

**Comment by mmmarcos** [comment]
> > #: ../../userguide/recovery_device.rst: these entries do not include the line number and do not appear in the rst file
> 
> Those texts comes from the alt of images. they would come back when you run make intl-fr

My bad, I thought it was a legacy text to remove. 
So if we keep those entries, why did you mark it fuzzy? shouldn't we fix translations?

> > #: ../../userguide/recovery_device.rst:20: these entries should be kept
> 
> I don't understand, we only have 1 entry for ../../userguide/recovery_device.rst:20

It was just an example of an entry containing a line number (`20`).

**Comment by FirelightFlagboy** [comment]
> So if we keep those entries, why did you mark it fuzzy?

It was added by `sphinx-intl` :shrug: 

> shouldn't we fix translations?

How do I fix that ?

**Pull request merged**